fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -57,6 +57,17 @@
       </xsl:otherwise>
         </xsl:choose>
     </field>
+
+    <-- JIRA TRAC-876 Define utk_mods_etd_author_orcid in Solr -->
+
+    <xsl:if test="mods:name[@type='orcid']">
+        <xsl:variable name="orcidtrim" select="substring-after(mods:name[@valueURI],'http://orcid.org/')"/>
```

Comment:
Your variable match is wrong:

<xsl:variable name="orcidtrim" select="substring-after(/mods:mods/mods:name/@valueURI,'http://orcid.org/')"/>

Reply:
Revised to correct path to /mods:mods/mods:name[@valueURI]